mockgen/tests/unexported_method/bugreport.go

```diff
@@ -0,0 +1,14 @@
+//go:generate mockgen -destination bugreport_mock.go -package bugreport -source=bugreport.go Example
+package bugreport
```

Comment:
This way `//go:generate` becomes the comment of the package. You can prevent that by inserting an empty line between `//go:generate` and the `package` declaration.